src/functionalTest/resources/scenarios/RIA-8719-internal-aip-late-remission-decision-partially-approved-out-of-country.json

```diff
@@ -0,0 +1,62 @@
+{
+  "description": "RIA-8719 Send letter notification to appellant for late remission decision (approved) - In country (internal case creation)",
```

Comment:
Change the description, it is out of country

Reply:
Good spot